Mathlib/LinearAlgebra/FreeModule/Basic.lean

```diff
@@ -85,6 +85,10 @@ def ChooseBasisIndex :=
   (exists_basis (R := R) (M := M)).some.1
 #align module.free.choose_basis_index Module.Free.ChooseBasisIndex
 
+/-- There is no hope of computing this, but we add the instance anyway to avoid fumbling with
+`open scoped Classical`. -/
+noncomputable instance : DecidableEq (ChooseBasisIndex R M) := Classical.decEq _
+
```

Comment:
Just drawing attention to the fact I adde this. We have a similar noncomputable decidable instance for `Real` since we know there will never be a computable one to compete with.